src/mmw/js/src/analyze/models.js

```diff
@@ -54,6 +54,33 @@ var AnalyzeTaskModel = coreModels.TaskModel.extend({
     }
 });
 
+var AnalyzeTaskCollection = Backbone.Collection.extend({
+    model: AnalyzeTaskModel
+});
+
+AnalyzeTaskCollection.getSingleton = function(App, aoi) {
```

Comment:
I think `getSingleton` needs to be defined outside of `Backbone.Collection.extend()` because it is instantiating its own collection.

`createViewModelCollection` doesn't need to be associated with `AnalyzeTaskCollection` at all, so I'll pull that out.